### PR TITLE
Deprecate MP objects

### PR DESCRIPTION
**Pull request by annakhm**
Signed-off-by: Anna Khmelnitsky <akhmelnitsky@vmware.com>

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -66,5 +66,8 @@ website-lint-fix:
 	@misspell -w -source=text website/
 	@terrafmt fmt ./website --pattern '*.markdown'
 
+website-list-category:
```
Comment (ksamoray) [inline GNUmakefile:69]: Is that related to the deprecation? Or just piggybacking?

**Comment by ksamoray** [comment]
data_source_nsxt_provider_info.go
resource_nsxt_vlan_logical_switch.go

Shouldn't these be deprecated too?

File: GNUmakefile (hunk 1 above)
Comment (annakhm) [reply GNUmakefile:69]: Sort of related - this verifies website categories are consistent. I'll add it to commit description

**Comment by annakhm** [comment]
> data_source_nsxt_provider_info.go resource_nsxt_vlan_logical_switch.go
> 
> Shouldn't these be deprecated too?

Thanks for noticing the vlan switch resource, added. provider info doesn't actually use MP api.

**Review by ksamoray — APPROVED** [review]
